release-note-script/README.md

```diff
@@ -0,0 +1,60 @@
+# Release Note Script
+
+These are scripts for generating the main body of release notes for Scalar products. There are two scripts available:
+
+- ReleaseNoteCreation.java ... Creates a release note body for a target repository.
+- MergeReleaseNotes.java ... Creates a merged release note body for ScalarDB (including community and enterprise edition).
+
+These scripts are invoked in GitHub Actions workflows when releasing a new version of Scalar products. 
+
+For the details of the release note please see [General rules]().
+
+## Prerequisites
+
+- [GitHub CLI](https://cli.github.com/)
+- Java 11+
+
+## How to Run the Scripts
+
+Note that primarily, these scripts are intended to be executed within GitHub Actions workflows.
+So usually you don't need to execute the scripts manually.
+
+### Create a Release Note Body for Each Repository
+
+*Assuming the use of Java 11*
+To run the script:
+
+```shell
+java --source 11 ReleaseNoteCreation.java <owner> <projectTitlePrefix> <version> <repository>
```

Comment:
Yes, I'd like to run this script using the Single-File Source-code feature of Java 11. Thus, you need to specify `--source 11` option.

Reply:
Sorry for the lack of clarity. My point is that if the `java` command version is 11 and the source code is compatible with Java 11, the single-file execution will work without `--source` as expected.

As far as I understand, the `--source` option is used to check if the source code is compatible with a different Java version from the `java` command being executed. Without the `--source` option, the `java` command checks the source code compatibility with the version of the `java` command being executed.

So, given that we require Java 11 as a prerequisite, we don't need to specify the `--source` option. (And, IMO, we shouldn't because, with the `--source` option, we expect the possibility of a different version of Java being executed.)

Reply:
Thank you for detailed point. I missed your point. And I checked the script can be execute without `--source 11` option. So remove the `--source` option form this comment and README as well. Thank you.

Reply:
Done. at 32d3fed